source/guides/core-concepts/cypress-studio.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Cypress Studio
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+
+- How to extend tests interactively using the Cypress Studio
+{% endnote %}
+
+# Overview
+
+Cypress Studio provides a visual way to generate tests within the Test Runner, by *recording interactions* against the application under test.
+
+
+{% imgTag /img/guides/cypress-studio/cypress-studio-overview.png "Cypress Studio" "no-border" %}
+
+## Using Cypress Studio
+
+The Cypress {% fa fa-github %} {% url "Real World App (RWA)" https://github.com/cypress-io/cypress-realworld-app %} will be use to demo the functionality of Cypress Studio.
```

Comment:
We should add a bit of introduction details into what the RWA is for the uninformed users. Ultimately we want them to know the RWA is the flagship demo app that we'll be using to demonstrate Cypress Studio.